*** tests/A-10-15-M-ok.out	2021-11-05 13:01:34.000000000 +0000
--- tests/A-10-15-M-ok.outhyp	2021-11-15 16:28:56.402289153 +0000
***************
*** 108,114 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|8
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 108,114 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|8
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 118,123 ****
--- 118,124 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ BARGAIN|SAL|8
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
