*** tests/A-15-09-M-ok.out	2021-10-31 23:52:14.000000000 +0000
--- tests/A-15-09-M-ok.outhyp	2021-11-19 16:25:20.742626431 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|R1|GARRAFA|9|362|362|0|ROLHA:9:18#SAL:45:720#VIDRO:90:900
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|R1|GARRAFA|9|362|362|0|ROLHA:9:180#SAL:45:1170#VIDRO:90:2700
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
